Userland/Libraries/LibGfx/Font/OpenType/Font.cpp

```diff
@@ -430,7 +430,7 @@ ErrorOr<NonnullRefPtr<Font>> Font::try_load_from_offset(ReadonlyBytes buffer, u3
         if (Checked<u32>::addition_would_overflow(table_offset, table_length))
             return Error::from_string_literal("Invalid table offset or length in font");
 
-        if (buffer.size() < table_offset + table_length)
+        if (buffer.size() <= table_offset + table_length)
```

Comment:
Is this right? If offset is 0 and the buffer is 1 byte large, and table_length is 1, shouldn't that be fine?

Reply:
You're right. The fuzzer test case I had had a `table_length` of 0, but that doesn't really make sense. I've reverted this change and added a `table_length` check above instead.